### ISSUE TITLE
RFC: disable Universal Viewer download, add a download link outside UV

### ISSUE THREAD

**Issue by andrewbenedictwallace** [UCLALibrary/ursus#201]
This is a request for comment – I'm making a suggestion / recommendation to see what people think.

*TLDR:* It will be hard to configure Universal Viewer's download menu the way we want. It would be easier to just disable downloads within UV and add a link to our single-item-page template.

- Universal Viewer configuration options are not documented, but a list is here: https://github.com/UniversalViewer/universalviewer/blob/master/src/extensions/uv-seadragon-extension/config/en-GB.json#L104
- It doesn't include the option to _always_ disable full-size downloads. The best we can do is set `maxImageWidth` to something like 1000. I assume (but haven't tested) that UV would then detect when the original is larger and disable that option. If we have any smaller items in the collection though, I assume the option would reimerge.
- Similarly, there's no way to directly disable the (currently broken) "Current view" option, but we could effectively set `currentViewDisabledPercentage` to something ridiculously low.
- Assuming both those fixes worked, we'd be left with a single download option (full-image, lower resolution), but the UV interaction would still involve bringing up a menu and selecting that option from it. Too many clicks.

# Recommended plan

- [ ] disable downloading from UV
- [ ] add a link to the non-UV portion of the single-item page, with the text "download" and a link to the lower-resolution version of the full image (maybe Kevin could help me work out the IIIF url...)
  - If it's easy to do, this would fit best [in the "Tools" section](https://github.com/projectblacklight/blacklight/wiki/Adding-new-document-actions).
  - If that takes much work, it might be easier to just squeeze it into the template somewhere else...

connected to https://github.com/UCLALibrary/ursus/issues/168
connected to https://github.com/UCLALibrary/ursus/issues/182

**Comment by bess** [comment]
+1 from me

**Comment by andrewbenedictwallace** [comment]
All implemented except https://github.com/UCLALibrary/ursus/issues/210; closing this one
